addons/l10n_uy/demo/account_demo.py

```diff
@@ -1,20 +1,490 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import api, models
+import logging
+from odoo import api, models, Command
+import time
+
+from odoo.exceptions import UserError, ValidationError
+_logger = logging.getLogger(__name__)
 
 
 class AccountChartTemplate(models.AbstractModel):
 
     _inherit = "account.chart.template"
 
+    @api.model
+    def _get_demo_data(self, company=False):
+        return {
```

Comment:
You should filter on the fact that `company.account_fiscal_country_id.code == "UY"`
Else, do the super()